frontend/src/app/main/data/workspace.cljs

```diff
@@ -1845,18 +1845,9 @@
                   tree-root            (get-tree-root-shapes pobjects)
                   only-one-root-shape? (and
                                         (< 1 (count pobjects))
-                                        (= 1 (count tree-root)))
-                  all-objects           (merge page-objects pobjects)
-                  comps-nesting-loop?   (not (->> (keys pobjects)
```

Comment:
Why do you remove this? Aren't those checks necessary now?

Reply:
They are including on the posterior call to 
`[parent-id frame-id]   (ctn/find-valid-parent-and-frame-ids candidate-parent-id page-objects (vals objects))`